change/@internal-react-composites-ad753f01-21b2-4c5d-8042-56413bdb161b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Update how Meeting composite deals with page state so it doe not subscribe to a new event every render.",
```

Comment:
nit typo: doe -> does

Also, this could be a bit shorter: "MeetingComposite: stop subscribing to state change event on every render"